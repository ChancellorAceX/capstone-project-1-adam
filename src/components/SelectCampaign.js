import React from 'react';

export default function SelectCampaign(props) {
  return (
    <option value={props.campaign[0]}>
      {`${props.campaign[1]}`}
    </option>
  );
}