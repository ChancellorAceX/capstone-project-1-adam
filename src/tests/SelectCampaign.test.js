import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SelectCampaign from '../components/SelectCampaign';

describe('SelectCampaign component', () => {
  const props = {
    "campaign": [
      6,
      "Fillory"
    ]
  };

  it('renders the full new campaign form component', () => {
    const wrapper = shallow(<SelectCampaign {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});