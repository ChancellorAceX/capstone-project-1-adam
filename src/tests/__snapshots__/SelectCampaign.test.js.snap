// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectCampaign component renders the full new campaign form component 1`] = `
<option
  value={6}
>
  Fillory
</option>
`;
